documentation/modules/proc-iot-creating-device.adoc

```diff
@@ -10,9 +10,11 @@ After installing the IoT services and creating an IoT project, you can create an
 
 ifeval::["{cmdcli}" == "oc"]
 :registry: $(oc -n enmasse-infra get routes device-registry --template='{{ .spec.host }}')
+:headers: -H 'Content-Type: application/json' -H "Authorization: Bearer ${TOKEN}"
```

Comment:
I think it would be good to have some kind of introduction of "why" and what this is being used for.

Reply:
Added a better explanation when exporting the token